File: Model/src/main/java/com/example/CellularNetwork.java
package com.example;

public record CellularNetwork(
        String networkType,    // 2G, 3G, 4G, 5G
        int signalStrength,    // dBm или ASU
        String mcc,
        String mnc,
        Integer cellId,
        Integer locationAreaCode
) {}

File: client/app/src/main/java/com/example/gpslessclient/model/CellularNetwork.kt
package com.example.gpslessclient.model


data class CellularNetwork(
    val networkType: String, // 2G, 3G, 4G, 5G
    val signalStrength: Int, // dBm или ASU
    val mcc: String?,
    val mnc: String?,
    val cellId: Int? = null,
    val locationAreaCode: Int? = null
)